app/locustio/jira/locustfile.py

```diff
@@ -3,7 +3,7 @@
     view_project_summary, view_dashboard, edit_issue, add_comment, browse_boards, view_kanban_board, view_scrum_board, \
     view_backlog, browse_projects
 from locustio.common_utils import ActionPercentage
-from extension.jira.extension_locust import custom_action
+from extension.jira.extension_locust import app_specific_action
 from util.conf import JIRA_SETTINGS
 
 action = ActionPercentage(config_yml=JIRA_SETTINGS)
```

Comment:
Could we rename actions in locustfile for jira in stile of confluence locustfile.